src/language-css/printer-postcss.js

```diff
@@ -679,6 +679,7 @@ function genericPrint(path, options, print) {
               )
             ])
           ),
+          shouldPrintTrailingComma(options.trailingComma, node) ? "," : "",
```

Comment:
How do you feel about refactoring this to `options.trailingComma && node.groups.length > 1` instead of a separate function?

Also, by testing the truthiness of `options.trailingComma`, instead of checking for `"all"`, we can print trailing commas if the value is `"es5"`.